nixos/modules/services/backup/borgbackup.nix

```
@@ -99,7 +99,17 @@ let
         BORG_REPO = cfg.repo;
         inherit (cfg) extraArgs extraInitArgs extraCreateArgs extraPruneArgs;
       } // (mkPassEnv cfg) // cfg.environment;
-      inherit (cfg) startAt;
+    };
+
+  mkBackupTimers = name: cfg:
+    nameValuePair "borgbackup-job-${name}" {
+      description = "BorgBackup job ${name} timer";
```

timers are correct now but aren't started automatically anymore :( gosh i hate computers sometimes, moving this one line was such a bad idea on the effort vs usefulness scale, so sorry for all the trouble :sweat: 
```suggestion
      description = "BorgBackup job ${name} timer";
      wantedBy = [ "timers.target" ];
```

---

thanks for taking so much time. initially i just wanted to have a persistent method, but i fell this is going into a very good direction.